README.md

```diff
@@ -1,9 +1,14 @@
-Good habits are difficult to start and difficult to keep. Even the most diligent among us struggle to maintain healthy habits. According to James Clear's book, *Atomic Habits*, the best way to begin a new habit is to anchor it to a habit or hobby you do regularly. This way, you naturally associate your new habit with something you enjoy, thus building a habit easily into your routine. And while this may be all the prompting you need to add a healthy habit to your schedule, we also know the benefits of having friendly accountability. Clear offers the idea of starting a habit with someone who already maintains your desired habit and the “Hawthorne Effect” shows us that productivity increases when people know they are being observed, so why not start a habit with someone else? 
-
-Our mobile app, HabitBuddy, provides a way for someone to start a new habit by matching them to another Buddy. Our goal is to connect Buddies with similar goals and encourage them to start a habit together. On our app, first-time Buddies fill out a basic profile and state a habit(s) they want to maintain. The Buddies will declare their new habit in the form of "habit stacking", a simple sentence format inspired by *Atomic Habits* that makes anchoring a new habit easy: After [current habit], I will [new habit]. While other habit-tracking apps boast motivating reminders and cool scheduling formats, HabitBuddy allows people to make a personal connection, not just with their habits, but with others who will inspire and motivate them. After the match, Habit Buddy provides basic contact information and lets the Buddies make the first move to connect. Because Buddies are also matched based on interests, like their favourite band or salty snack, they have another opportunity to anchor their new habit to their Buddy and their shared interest.
+Good habits are difficult to start and difficult to keep. Even the most diligent among us struggle to maintain healthy habits. According to James Clear's book, *Atomic Habits*, one of the most effective ways to start a new habit is to anchor it to an activity or interest you already do on a regular basis, which he lays out in detail through four laws: make it obvious, make it easy, make it attractive, and make it satisfying. 
+ 
+Rooted in these ideas, our mobile app, Habit Buddy, a visionary project to help you start a new habit. By matching you to someone with similar goals and interests, Habit Buddy sets you both up for success and reward by giving you the means to create accountability and encourage each other as you work towards your personal and shared goals. Habit Buddy makes it easy and fun to start and maintain a habit by following Clear’s four laws: 
+  1. Make it obvious. Habit Buddy makes your habit obvious by sending reminders and giving you the opportunity to contact a Buddy. After filling out a simple profile, you’ll be able to easily set up accountability with your Buddy and share your habit goal with each other. 
```

Comment:
Make it obvious. **Small changes in your environment can lead you to make large changes in your lifestyle.** Habit Buddy makes your habit obvious....